src/main/java/org/jitsi/impl/protocol/xmpp/ChatMemberImpl.java

```diff
@@ -229,7 +229,8 @@ void processPresence(Presence presence)
         {
             boolean[] startMuted = { ext.getAudioMuted(), ext.getVideoMuted() };
 
-            if (getRole().compareTo(MemberRole.MODERATOR) < 0)
+            // XXX Is this intended to be allowed for moderators or not?
```

Comment:
It was moderator before, yes? Do you think this is wrong?

Reply:
It looked suspicous to me, because we allow moderators to mute people, etc. So I thought the intention was to use "<=", and we never caught it because we have `allowners` in most places.

In the end I kept the code but removed the comment, because the new code reads clearly and because I hope all this "start muted" code is going to go away soon.